www/includes/easyparliament/templates/html/static/privacy.php

```diff
@@ -13,15 +22,130 @@
             order to run the website.</li>
             </ol>
 
+            <p>If you’d like to know more, read on.</p>
+
+            <h3>What information we collect and how we use it</h3>
+
+            <p><strong>If you sign up to receive alert emails</strong>, we
+            collect your email address along with the name of the MP, postcode,
+            or the keywords to which you subscribe.</p>
+            <p>We’ll send you an email every time your chosen keyword/s are
+            spoken in a parliament or assembly we cover, or every time your
+            chosen MP speaks. At most (depending on how common your keyword is,
+            and how often your MP speaks), these will come at a frequency of one
+            a day and only while a parliament or assembly we cover is in session.</p>
+            <p>We may analyse alert sign-up data in aggregate, to understand
+            volumes and trends in what interests citizens in parliamentary
+            proceedings. If so, the data is anonymised, ie, stripped of all
+            information which would identify a specific user. Our Research Data
+            Release policy may be seen on request.</p>
+
+            <p><strong>If you register with the site</strong>, we collect your
+            full name and email address, and — if you opt to provide them — your
+            postcode and the URL of your website.</p>
+            <p>We do not use this data for any purpose other than to allow you
+            to log in to the site.</p>
+
+            <p><strong>If you choose to receive email updates</strong>, we add
```

Comment:
This para (and unsubscribe section below) not needed if #1358 done.

Reply:
+1 – happy to remove if @MyfanwyNixon @crowbot agree. At the very least, we’s need to amend the wording of this to make it clear that the "email updates" we’re talking about here are the email _newsletter_ you join via the site footer, and not the "email alerts" you sign up to in various other parts of the website.

Reply:
Not needed, but it's nice to cover everything IMO. Agree that 'update' would be better as 'newsletter'.

Reply:
Ok. Updated the wording here, but kept the section.